packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -871,9 +888,9 @@ const AccountSwitcher = props => {
                 </Text>
                 <Text size='xs' color='prominent' className='acc-switcher__balance'>
                     <Money
-                        currency={isRealAccountTab ? props.obj_total_balance.currency : 'USD'}
+                        currency={props.obj_total_balance.currency}
```

Comment:
As we are supposed to show the total assets of virtual accounts based on their location's currency, we will not need this condition anymore.

Reply:
Does this mean for virtual account, we support different currencies?

Reply:
The currency that is shown for demo account was always USD while we want to show the total assets based on the currency of the user's location.